Automattic/sensei
# Add setup wizard Features step - with mocked data

The first part of #3044 (Frontend with mocked data)

### Changes proposed in this Pull Request

* Add setup wizard Features step.
* For this PR the data is mocked.

### Testing instructions

* Go to `/wp-admin/admin.php?page=sensei_onboarding&step=features`.
* Click on "Continue" without mark any item. You should be redirected to the next step.
* Go back to `/wp-admin/admin.php?page=sensei_onboarding&step=features`.
* Mark at least one checkbox and click on "Continue".
* In the modal, click on "I'll do it later". You should be redirected to the next step.
* Go back to `/wp-admin/admin.php?page=sensei_onboarding&step=features`.
* Select all items. And click on "Continue".
* In the modal click on "Install now". You should see the items with their status and an "Installing..." button with no action.
* Click on "Back to optional features". You should return to the selection screen with the same checkboxes marked.
* Mark the "Course progress" and "Certificates" and click on "Continue".
* You should see the buttons "Retry" (No action for this PR) and "Continue" because there are some items with error.
* If you click on "Continue" you should to the next step.
* Go back to `/wp-admin/admin.php?page=sensei_onboarding&step=features`.
* Mark only the "Course Progress" option and click on "Continue".
* In the modal click on "Install now". You should see the items with the button "Continue" because all the items are completed with success.
* Click on "Continue" and you should go to the next step.

### Caveats

To make sure that the buttons will appear correctly, you need to install the [Gutenberg plugin](https://wordpress.org/plugins/gutenberg) in the latest version. Related issue: https://github.com/Automattic/sensei/issues/3173
